lib/rubocop/cop/style/script_permission.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks whether the file permission is executable.
```

Comment:
This should be extended a bit - after all the cop applies just scripts. Frankly, from this description people looking the manual would be quite confused.